process/graduation_criteria.md

```diff
@@ -0,0 +1,115 @@
+maturity level, each previous level will receive a vote until a 2/3 supermajority is met. If the project does not receive 
+enough votes at the Sandbox stage, the project is rejected.
+
+Each project in AzCU will participate in a quarterly review by the TOC. The review will assess the project's
+maturity level as well as its community momentum. A 2/3 supermajority vote is required to graduate to the next maturity
+level. If the project is no longer needed or has become obsolete, the TOC can also vote with a 2/3 supermajority to archive
+the project.
+
+For a project that has reached the "Graduation Stage" and would benefit from a more neutral home in a community foundation,
+the option of project donation is an option.
+
+![AzCU Maturity Process](../images/maturity-diagram.png)
+
+## Sandbox Stage
+Sandbox projects are "early stage" projects that warrant experimentation. The project should be beneficial to Azure and
+the Cloud Native community, open to contributions from the public and have well-founded aspiration.
+
+The AzCU Sandbox is the entry point for early stage projects and has the following goals:
+- Encourage public visibility of experiments or other early work that can add value to the mission of AzCU, Cloud Native
+  in Azure and its impact on the larger Cloud Native community.
+- Facilitate alignment with existing projects within Azure as well as the larger community
+- Remove obstacles to adoption and contribution by ensuring projects adhere to Microsoft Open Source Guidelines, code of
+  conduct and IP policy requirements
+  
+### Entry Requirements
+- Required backing of at least one member of the product management
+- Display of the AzCU "Sandbox" badge on website / readme
+- Draft project proposal for review and approval by TOC
+  - A great example is the [CNCF Project Proposal Process](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/project_proposals.adoc)
+  - TODO: create our own project proposal process based on the CNCF project proposal process
+- Begin working toward achieving [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/)
+  - Must have 
+    - Gated PRs with unit tests
+    - MSFT code of conduct
+    - LCA bot enabled
+
+## Incubating Stage
+An Incubating project is one that has met all of the requirements of the Sandbox stage and is showing accelerating 
+adoption.
+
+### Entry Requirements
+- Minimum of 3 customers using the project in a production or pre-production environment
+- Minimum of 2 maintainers
+- Full due diligence of the TOC
+  - A great example is the [CNCF Due Diligence Guidelines](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/due-diligence-guidelines.md)
+  - TODO: create our own DD guidelines based on the CNCF DD guidelines
+- Demonstrate a substantial ongoing flow of commits and merged contributions
+- Have a healthy number of issues and pull requests from external contributors
```

Comment:
what's a "healthy" number?

Reply:
the CNCF guidelines say "Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria" - I think that applies here too

Reply:
IDK. I'm hoping folks may have opinions :).

Reply:
Is it preferable to measure based on the % of issues/PRs that originate from external vs internal contributors?

Reply:
Might be better defined as a healthy ration `issues:PRs` or `issues:Planned releases`